src/libraries/Directory.Build.targets

```diff
@@ -272,4 +278,21 @@
       </ItemGroup>
     </When>
   </Choose>
+
+  <Target Name="LibIntellisenseDocs" Condition="'$(IncludeBuildOutput)' == 'true'">
+   <ItemGroup>
+      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)1033\$(TargetName).xml" Condition="Exists('$(XmlDocFileRoot)1033\$(TargetName).xml')">
+        <PackagePath>lib/$(TargetFramework)</PackagePath>
+      </TfmSpecificPackageFile>
```

Comment:
nit:

```suggestion
      <TfmSpecificPackageFile Include="$(XmlDocFileRoot)1033\$(TargetName).xml"
                              Condition="Exists('$(XmlDocFileRoot)1033\$(TargetName).xml')"
                              PackagePath="lib/$(TargetFramework)" />
```

Reply:
hard to apply the suggestion with the right whitespaces. Hope this is correct :D